operator/Makefile

```diff
@@ -10,7 +10,7 @@ all: manager
 
 # Run tests
 test: generate fmt vet manifests
-	go test ./api/... ./controllers/... -coverprofile cover.out
+	go test ./api/... ./controllers/... ./chartdownloader/... -coverprofile cover.out
```

Comment:
This wasn't even running in CI 😬

Reply:
I think we should just use `go test ./...` , so we don't have to keep adding new pkgs

Reply:
Sold